Marlin/src/pins/pins_BIGTREE_SKR_MINI_V1_1.h

```diff
@@ -123,10 +126,13 @@
 // SD Card
 //
 
-// Marlin uses the SD drive attached to the LCD
+// By default the onboard SD is enabled. 
+// To disable it and use an external SD (connected to LCD)
+// enable STM32_SD_LCD.
+
 //#define STM32_SD_LCD
 
-#ifdef STM32_SD_LCD
+#if ENABLED(STM32_SD_LCD)
   #define SD_DETECT_PIN    PB9
   #define ENABLE_SPI3
```

Comment:
#define ENABLE_SPI3 
  should be 
  #define DISABLE_JTAG

Reply:
Since that is already always the case, I guess the line may be removed entirely.

Reply:
> #define ENABLE_SPI3
> should be
> #define DISABLE_JTAG

@Msq001 Why do you say that? #define DISABLE_JTAG is already defined at the top of pins_BIGTREE_SKR_MINI_V1_1.h. > #define ENABLE_SPI3 determines which SPI device to use and which set of SPI pins to use in \HAL_STM32F1\spi_pins.h

#if ENABLED(ENABLE_SPI3)
  #define SPI_DEVICE 3
  #define SCK_PIN   BOARD_SPI3_SCK_PIN
  #define MISO_PIN  BOARD_SPI3_MISO_PIN
  #define MOSI_PIN  BOARD_SPI3_MOSI_PIN
  #define SS_PIN    BOARD_SPI3_NSS_PIN
#elif ENABLED(ENABLE_SPI2)
  #define SPI_DEVICE 2
  #define SCK_PIN   BOARD_SPI2_SCK_PIN
  #define MISO_PIN  BOARD_SPI2_MISO_PIN
  #define MOSI_PIN  BOARD_SPI2_MOSI_PIN
  #define SS_PIN    BOARD_SPI2_NSS_PIN
#else
  #define SPI_DEVICE 1
  #define SCK_PIN   BOARD_SPI1_SCK_PIN
  #define MISO_PIN  BOARD_SPI1_MISO_PIN
  #define MOSI_PIN  BOARD_SPI1_MOSI_PIN
  #define SS_PIN    BOARD_SPI1_NSS_PIN
#endif

Reply:
Take comments from random users with a grain of salt.